modules/managers/galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/test/java/dev/galasa/zos3270/spi/TerminalTest.java

```diff
@@ -45,5 +50,106 @@ public boolean connectClient() throws NetworkException {
         textScan, codePage, network);
 
     }
+
+    @Test
+    public void testConvertNullTerminalToJsonReturnsNull() throws Exception {
```

Comment:
The name of this method confused me a little, as it doesn't appear that the Terminal itself is null, rather the screen should be at this point? I'm not certain though. From L75 it looks to me like Terminal is not null.

Reply:
Agreed, it's a bit confusing because we have a Terminal class which represents the actual terminal and its functionality, but we also have another Terminal class that's just a pojo representing a terminal screen 😅 

Addressed in https://github.com/galasa-dev/galasa/pull/403/commits/17e34c1ca77f045b63db3909d5f5eeba45654f72